lib/shared/utils.js

```diff
@@ -48,22 +48,15 @@ function timecodeToMicroseconds(timecode, fps) {
 }
 
 function extractStyling(text) {
-  let unformattedText = '';
-  if (text.search(/<br>/) && !text.startsWith('<br>')) {
-    unformattedText = text.replace(/<br>/, '\n');
-  }
-  const formattingTags = unformattedText.match(/<[^>]*>|{[^}]*}/g);
-  if (formattingTags && formattingTags.length) {
-    formattingTags.forEach(tag => {
-      if (tag.search(/{[^}]*}/) > -1) {
-        unformattedText = unformattedText.replace(tag, ' ').replace(/\s\s+/, ' ');
-      } else {
-        unformattedText = unformattedText.replace(tag, '');
-      }
-    });
-  }
-
-  return unformattedText.trim();
+  const regexReplace = [
+    [/(?<!^)<br>/g, '\n'],
```

Comment:
Unfortunately negative look behinds are not supported in all browsers.  Although it works locally, we have previously run into issues using negative look behinds on production.  Is there another way to write the regex here and on line 55?